src/cps/meson.build

```diff
@@ -6,6 +6,7 @@ conf.set_quoted('CPS_VERSION', meson.project_version())
 foreach f : ['unreachable']
   conf.set10('CPS_USE_BUILTIN_@0@'.format(f.to_upper()), cpp.has_function(f))
 endforeach
+conf.set_quoted('CPS_LIBDIR', get_option('libdir'))
```

Comment:
I don't love this and was a bit hesitant to add it since I don't think it should be used outside of testing. The more I think about it the less I like it and the more I'd rather have a different solution.

Reply:
Okay, I've completely dropped this patch, and instead we're using CPS_PATH instead of CPS_PREFIX_PATH. This gives us more robust testing since the libdir is now irrelevant. It does mean that we need some tests for CPS_PREFIX_PATH.

Reply:
And that found some interesting bugs.